source/nodejs/adaptivecards/src/utils.ts

```diff
@@ -272,3 +272,21 @@ export function addCancelSelectActionEventHandler(element: HTMLElement) {
         e.cancelBubble = true;
     };
 }
+
+export function SetDateOrTimePickerVisibility(input: HTMLInputElement, value: string | undefined, show: boolean) {
```

Comment:
What is the `value` parameter? The function name doesn't seem to call for a value and doesn't make it clear what it is for.